concrete/blocks/youtube/form_setup_html.php

```diff
@@ -88,7 +88,7 @@
                                 $disabledattr['disabled'] = 'disabled';
                             }
                             echo $form->checkbox('modestbranding', 1,  (isset($modestbranding) ? $modestbranding : true), $disabledattr); ?>
-                            <?php echo t("Hide YouTube Logo")?>
+                            <?php echo t("Hide YouTube logo")?>
```

Comment:
Any reason to change the case? Translators will have to re-translate this string again...

Reply:
I changed the case to match the other labels. If it creates a lot of extra work, I can change it back.

Reply:
Not that much work I guess: the transition system should suggest the previous translations